packages/rancher-istio/charts/README.md

```diff
@@ -6,7 +6,7 @@ A Rancher created chart that packages the istioctl binary to install via a helm
 
 ### pre-requisites
```

Comment:
nit:
```suggestion
### Requirements
```

Reply:
nit: 

Also, if we are updating the README, on the later parts of the `README.md`, installation should be capitalized and the helm install instruction should be encapsulated with a ``` like below:

```bash
helm install rancher-istio ./ --create-namespace -n cattle-istio-system
```